index.html

```diff
@@ -151,29 +151,31 @@ <h2>
     </section>
     <section>
       <h2>
-        Interface definitions
-      </h2>
-      <section>
-        <h2>
-          Extensions to the <a>Screen</a> interface
-        </h2>The CSSOM View specification defines a <code>Screen</code>
-        interface [[!CSSOM-VIEW]], which this specification extends:
-        <pre class='idl'>
+        Extensions to the <a>Screen</a> interface
+      </h2>The <a data-cite="CSSOM-View">CSSOM View Module</a> specification
+      defines a <code>Screen</code> interface, which this specification
+      extends:
+      <pre class='idl'>
           partial interface Screen {
             [SameObject] readonly attribute ScreenOrientation orientation;
           };
         </pre>
+      <section>
+        <h2>
+          <dfn data-dfn-for='Screen'>orientation</dfn> attribute
+        </h2>
         <p>
-          The <dfn data-dfn-for='Screen'>orientation</dfn> object is an
-          instance of <a>ScreenOrientation</a>, which is described below.
+          The <a>orientation</a> attribute is an instance of
+          <a>ScreenOrientation</a>, which is described below.
```

Comment:
Probably ok to drop the "which is described below", as it's hyperlinked. 

```suggestion
          <a>ScreenOrientation</a>.
```